src/components/pagination/examples/pagination-no-range-indicator/index.njk

```diff
@@ -7,32 +7,32 @@
         "hideRangeIndicator": true,
         "pages": [
             {
-                "url": "#"
+                "url": "#page1"
             },
             {
-                "url": "#",
-                "current": true
+                "url": "#page2"
             },
             {
-                "url": "#"
+                "url": "#page3"
             },
             {
-                "url": "#"
+                "url": "#page4"
             },
             {
-                "url": "#"
+                "url": "#page5"
             },
             {
-                "url": "#"
+                "url": "#page6"
             },
             {
-                "url": "#"
+                "url": "page7",
+                "current": true
             },
             {
-                "url": "#"
+                "url": "#page8"
             },
             {
-                "url": "#"
+                "url": "#page9"
             },
             {
                 "url": "#"
```

Comment:
Do we not want to set the page number urls for all the nodes? Other than that looks good

Reply:
Yeah we can do. It was more to test that next/prev links rendered the correct url. Obviously the url's don't go anywhere.

Reply:
Yeah just makes sense to me if you're going to do it for some you may as well do it for all of them